packages/fxa-content-server/app/scripts/views/sign_up_password.js

```diff
@@ -78,12 +77,10 @@ const SignUpPasswordView = FormView.extend({
     );
   },
 
-  isValidEnd() {
-    if (!this._doPasswordsMatch()) {
-      return false;
-    }
+  onFormChange() {
```

Comment:
This was the fix for https://github.com/mozilla/fxa/issues/3489, for some reason having the `keyup #vpassword` event caused the password info bubble to disappear. Instead, I hooked into the existing events that get trigger when form element changed.